EuclideanGeometry/Foundation/Axiom/Basic/Plane.lean

```diff
@@ -47,35 +47,75 @@ instance : AddTorsor (Vec) P := by infer_instance
 
 /- vector $AB +$ point $A =$ point $B$ -/
 @[simp]
-theorem start_vadd_vec_eq_end (A B : P) : (VEC A B) +ᵥ A = B := sorry
+theorem start_vadd_vec_eq_end (A B : P) : (VEC A B) +ᵥ A = B := by
+  rw [Vec.mk_pt_pt]
+  exact vsub_vadd B A
```

Comment:
Here `rw [Vec.mk_pt_pt]` only unfold the definition. You can directly write `:= by exact vsub_vadd B A` to show this theorem, or even shorter `:= vsub_vadd B A`.

`simp` do need a `rw` to unfold the definition. But `exact` don't need it.